### ISSUE TITLE
Handled exception: ValueError: Internal group has more than one project assigned!

### ISSUE THREAD

**Issue by burner1024** [WeblateOrg/weblate#7503]
### Describe the issue

Trying upgrade to 4.11.2.
Turns out, I have a group which has no projects assigned. Migrations fail, container doesn't start.
No option to delete the group in UI. Don't see a management command for it, either.

Migration [checks](https://github.com/WeblateOrg/weblate/blob/9faf268eb10021098986a56e57a5b733b0661791/weblate/auth/migrations/0021_migrate_internal_groups.py#L17) for exactly one group, but error message says "more than one".

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

Standard docker-compose up with new version.

### Expected behavior

1. Upgrade not failing when a group has no projects assigned.
2. Some option to delete the group.
3. Less misleading error message.

### Screenshots

Screenshots are from previous 4.9 version.

![Captura de pantalla de 2022-04-16 18-02-38](https://user-images.githubusercontent.com/1279852/163672527-a9e40dd0-9c6f-4119-bd36-f431b4a4a143.png)

![Captura de pantalla de 2022-04-16 17-51-45](https://user-images.githubusercontent.com/1279852/163672479-3f58957e-240f-4587-b259-d96d01b99be4.png)

### Exception traceback

```pytb
[2022-04-16 10:34:13,918: WARNING/45] Handled exception: ValueError: Internal group Dangerous Quest!@Administration has more than one project assigned!
Traceback (most recent call last):
  File "/usr/local/bin/weblate", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/site-packages/weblate/runner.py", line 34, in main
    utility.execute()
  File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 414, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 460, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 98, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/commands/migrate.py", line 290, in handle
    post_migrate_state = executor.migrate(
  File "/usr/local/lib/python3.10/site-packages/django/db/migrations/executor.py", line 131, in migrate
    state = self._migrate_all_forwards(
  File "/usr/local/lib/python3.10/site-packages/django/db/migrations/executor.py", line 163, in _migrate_all_forwards
    state = self.apply_migration(
  File "/usr/local/lib/python3.10/site-packages/django/db/migrations/executor.py", line 248, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/usr/local/lib/python3.10/site-packages/django/db/migrations/migration.py", line 131, in apply
    operation.database_forwards(
  File "/usr/local/lib/python3.10/site-packages/django/db/migrations/operations/special.py", line 193, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/usr/local/lib/python3.10/site-packages/weblate/auth/migrations/0021_migrate_internal_groups.py", line 18, in migrate_internal_groups
    raise ValueError(
ValueError: Internal group Dangerous Quest!@Administration has more than one project assigned!
  Applying weblate_auth.0021_migrate_internal_groups...
```

### How do you run Weblate?

Docker container

### Weblate versions

Standard 4.11.2-1 Docker.

### Weblate deploy checks

```shell
Probably not relevant, but if needed, can add.
```

**Comment by nijel** [comment]
You've probably made some modifications to Weblate internal groups, and the migration does not deal with that. Please fix the project assignment for the `Dangerous Quest!@Administration` group manually before doing the migration.

**Comment by nijel** [comment]
See also https://github.com/WeblateOrg/docker/issues/1454

**Comment by burner1024** [comment]
I still think there's at least 3 issues here.

**Comment by Vadsher** [comment]
I faced with the same issue when updating Weblate from 4.10.1-1 to 4.11.2-1. Weblate runs in Docker container. Updated using pull and deploy new image
As mentioned above there is no option to delete the group in UI. I tried to fix the projects assignment:

> weblate@c7da50344aa7:/$ weblate shell -c 'from weblate.auth.models import Group; group = Group.objects.get(name="web_app_v2@Translations_All"); print(group.projects)'
> trans.Project.None

But there are only one entry and I couldn't find any mention about "trans.Project.None" in any my projects (also we have only two projects).
Maybe someone can give a good idea where to find the solution or knows how to solve it. Thank you

**Comment by burner1024** [comment]
So you want to delete or fix?
[API](https://docs.weblate.org/en/latest/api.html#delete--api-groups-(int-id)-) should work for deletion.

**Comment by Vadsher** [comment]
First of all, thank you for responding. 
1. I would like to understand what "trans.Project.None" is.
2. We have only 2 projects, one is inactive, but we can't delete it.
3. "web_app_v2@Translations_All" internal group is used in our active project and I don't want to damage anything in any way.

Therefore:
1. The ideal option is to fix it. As far as I understand, we need to remove the binding to the first inactive project.
2. If I delete something, I want to understand what it is.

If you need any more information, please let me know.
P.S. Weblate is very interesting program, I encountered it for the first time, I am still figuring it out.

**Comment by burner1024** [comment]
I think your error says that no projects are assigned.
Although I'm not sure how Translations_All came to be an internal group in the first place, did you rename Administration or something?

From my experience, it's best to stay away from internal groups. With Weblate's limitations and quirks, they are less than useless. Just define your own and grant permission through those.

**Comment by Vadsher** [comment]
I forgot to say that I originally updated the image from 4.8-1 to 4.10.1-1, the migration of the database was successful.

I definitely did not rename the group, maybe someone before me. But the group "web_app_v2@Administration" is present. In the history of the group "web_app_v2@Translations_All" I found a record that it was added through the Weblate administration portal.

> I think your error says that no projects are assigned

I thought so too. 

In that case, what's the best thing to do first:
- try adding to the active project? But I have to mention, UI changes are not available, and through API or shell I still can't figure out how to do it correctly.
- is it possible to remove the property of the internal group? What can it lead to?
